ipykernel/zmqshell.py

```diff
@@ -33,7 +33,7 @@
 from IPython.core.magics import MacroToEdit, CodeMagics
 from IPython.core.magic import magics_class, line_magic, Magics
 from IPython.core import payloadpage
-from IPython.core.usage import default_gui_banner
+from IPython.core.usage import default_banner
```

Comment:
Sorry, I was thinking that a new release of `ipykernel` would be hold until IPython 5.0 is ready.

But you're right, `default_banner` is present in previous IPython releases, so there's no problem with this change :-)